src/components/UserForm/BasicInformationSection.test.js

```diff
@@ -0,0 +1,219 @@
+    beforeEach(() => {
+        useDataQuery.mockReset()
+        useFeatureToggle.mockReset()
+    })
+
+    it('does not show email verification message when system is v41 or earlier', () => {
+        useFeatureToggle.mockReturnValue({ displayEmailVerifiedStatus: false })
+        useDataQuery.mockReturnValue({ data: { enforceVerifiedEmail: false } })
+
+        const user = {
+            emailVerified: false,
+        }
+
+        renderWithForm(
+            <BasicInformationSection
+                user={user}
+                inviteUser={''}
+                userInterfaceLanguage="en"
+                interfaceLanguageOptions={[]}
+                userDatabaseLanguage="en"
+                databaseLanguageOptions={[]}
+                currentUserId="1"
+            />
+        )
+        expect(
+            screen.queryByText(/this user email has been verified/i)
+        ).toBeNull()
+        expect(
+            screen.queryByText(/this user does not have a verified email/i)
+        ).toBeNull()
+    })
+
+    it('does not show email verification message when system is v42 or later and a system does not have email configured', () => {
+        useFeatureToggle.mockReturnValue({ displayEmailVerifiedStatus: true })
+        useDataQuery.mockReturnValue({ data: { enforceVerifiedEmail: false } })
+
+        renderWithForm(
+            <BasicInformationSection
+                user={null}
+                inviteUser={''}
+                userInterfaceLanguage="en"
+                interfaceLanguageOptions={[]}
+                userDatabaseLanguage="en"
+                databaseLanguageOptions={[]}
+                currentUserId="1"
+            />
+        )
+        expect(
+            screen.queryByText(/this user email has been verified/i)
+        ).toBeNull()
+        expect(
+            screen.queryByText(/this user does not have a verified email/i)
+        ).toBeNull()
+    })
+
+    it('does not show email verification message when creating a new user (no user data available)', () => {
+        useDataQuery.mockReturnValue({ data: { enforceVerifiedEmail: false } })
+        useFeatureToggle.mockReturnValue({ displayEmailVerifiedStatus: true })
+
+        renderWithForm(
+            <BasicInformationSection
+                user={null} // No user data, so it’s creating a new user
+                inviteUser={'INVITE_USER'}
+                userInterfaceLanguage="en"
+                interfaceLanguageOptions={[]}
+                userDatabaseLanguage="en"
+                databaseLanguageOptions={[]}
+                currentUserId="1"
+            />
+        )
+        expect(
+            screen.queryByText(/this user email has been verified/i)
+        ).toBeNull()
+        expect(
+            screen.queryByText(/this user does not have a verified email/i)
+        ).toBeNull()
+    })
+
+    it('shows email error message when system is v42 or later and a system does not have email configured', async () => {
```

Comment:
if system does not have email configured, these messages should not show, so again I think the test naming is off? Maybe:`shows email error message when system is v42 or later and a user does not have a verified email`